common/reviews/api/node-library-build.api.ts

```diff
@@ -1,13 +1,9 @@
 export declare function addSuppression(str: string): void;
 
 class ApiExtractorTask extends GulpTask<IApiExtractorTaskConfig> {
-  // (undocumented)
```

Comment:
The CI build won't catch it because the web-build-tools uses the last published version to generate the .api.ts files, not the currently checked in version.